ns1/config.go

```diff
@@ -54,7 +55,11 @@ func (c *Config) Client() (*ns1.Client, error) {
 		client = ns1.NewClient(httpClient, decos...)
 	}
 
-	client.RateLimitStrategySleep()
+	if parallelism := c.RateLimitParallelism; parallelism > 0 {
+		client.RateLimitStrategyConcurrent(parallelism)
+	} else {
+		client.RateLimitStrategySleep()
+	}
```

Comment:
I see the old code didn't do this, but might be cleaner to do this by appending `ns1.SetRateLimitFunc()` to `decos`, instead of modifying the client directly, since that would mirror how we set the API key and endpoint.

Reply:
yeah, we could be more consistent with it, but for now, I prefer to leave it this way. It would be a bit of a refactor to pull the actual ratelimitfuncs out of their wrappers (to pass to the setter func), and I could see it breaking existing users, unless we keep the current style working, so we wouldn't really end up simplifying anything.